### DISCUSSION TITLE
Realtime Not Working? (Asking for help)

### DISCUSSION THREAD

**Discussion by wcooperscott** [supabase/supabase#7861] [Questions] [answered]
Using Supabase JS (with Angular) to do the following:

`this.supabase.supabase.from('song')
.on('*', (load) => console.log(load))
.subscribe(status => console.log(status));`

I have enabled replication for `supabase_realtime` and toggled on realtime for the the `song` table.
I have also set the song replica identity to FULL.
I have restarted the AWS server.
My subscription status reads out  `SUBSCRIBED` on connection.
I have disabled all RLS for every table.
I removed any reference links in the tables.
I deleted the project and recreated it (and re-enabled `supabase_realtime` + realtime on `song` table)

I am making changes to the table rows from the Supabase website's graphical SQL interface.

**However, I do not receive any events/callbacks.**
No insert, update, or delete events are executed (the callback **never** fires).
No errors are given in the console.

If you have any ideas or suggestions, **please post them**. At this point, anything helps.

**Comment by felibunnyy** [comment]
can consider turning off StrictMode:
https://github.com/supabase/realtime-js/issues/169

**Reply by wcooperscott** [reply]
I am using Angular, not React.
I am not familiar with React but I believe `StrictMode` is a feature of react?

Regardless, I appreciate the suggestion.

**Comment by GaryAustin1** [comment, marked as answer]
Please contact support.  There was an update to some instances with a version of realtime that is not working.  Only they can tell you if that is the issue.
Your subscription will look normal (websocket running) but if you go to realtime.subscriptions table there will be no rows.

**Reply by wcooperscott** [reply]
Executing `select * from realtime.subscription;` on the website app's `SQL Editor` returns `Success. No rows returned` as you mentioned.

Best method to go about it is to email [support@supabase.io](mailto:support@supabase.io)?
Sending them an email now. Thank you @GaryAustin1

**Reply by GaryAustin1** [reply]
Best method is to use help button in your instance at top of dashboard as they will need your instance url, but if you provide that in email they should be able to look.

**Reply by wcooperscott** [reply]
Stas, from support, got back to me. This was the issue and they where able to fix it. Thank you 🎉
